### ISSUE TITLE
Error starting multiples tasks using $restVar

### ISSUE THREAD

**Issue by marcospauloz** [aelassas/Wexflow#146]
When we tried to start 2 or more tasks at the same time, Wexflow uses $restVar1 instead of the value.
It's the same TaskId and the same $restVar1, but with different values in each request.

Thanks for your attention!

**Comment by aelassas** [comment]
@marcospauloz can you post your workflow and your payload?

**Comment by marcospauloz** [comment]
Sure!
This is the Workflow

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Workflow xmlns="urn:wexflow-schema" id="149" name="DeliveryFiles" description="DeliveryFiles">
   <Settings>
      <Setting name="launchType" value="trigger" />
      <Setting name="enabled" value="true" />
      <Setting name="approval" value="false" />
   </Settings>
   <LocalVariables />
   <Tasks>
      <Task id="1" name="Http" description="Downloading files" enabled="true">
         <Setting name="url" value="$restVar1" />
      </Task>
      <Task id="2" name="FilesMover" description="Moving files to Http folder" enabled="true">
         <Setting name="selectFiles" value="1" />
         <Setting name="destFolder" value="C:\App\temp" />
      </Task>
      <Task id="3" name="FilesLoader" description="Loading Zip..." enabled="true">
         <Setting name="file" value="C:\App\temp\$restVar2" />
      </Task>
      <Task id="4" name="Unzip" description="Extracting ZIP archives" enabled="true">
         <Setting name="selectFiles" value="3" />
         <Setting name="destDir" value="$restVar3" />
         <Setting name="createSubDirectoryWithDateTime" value="false" />
      </Task>
   </Tasks>
</Workflow>
```

And this is the request body:

```json
{
    "WorkflowId": 149,
    "Variables": [
        {
            "Name": "restVar1",
            "Value": "http://sw16-25/api/buildjobs/p5798o63b1h000ea/artifacts/token/af57ec0d27e515d8d3b3739193d40f3a4e8bbc8a890618f3757ab3bca5d084f0/Web/ATS.UI.Web/bin/ATSWeb.zip"
        },
        {
            "Name": "restVar2",
            "Value": "ATSWeb.zip"
        },
        {
            "Name": "restVar3",
            "Value": "C:\\App\\Web"
        }
    ]
}
```

**Comment by aelassas** [comment]
@marcospauloz issue reproduced and fixed. I'll publish the fix in the v5.1.
